src/generic/stage1/json_string_scanner.h

```diff
@@ -35,9 +37,12 @@ class json_string_scanner {
 public:
   really_inline json_string_block next(const simd::simd8x64<uint8_t> in);
   really_inline error_code finish(bool streaming);
+  really_inline bool in_unclosed_string() { return prev_in_string; }
```

Comment:
So you have wrapped a variable in a really_inline function? Why?

Reply:
I think it had to do with wanting a different name. It's only used internally, so it doesn't seem like there's any good reason. I'll get rid of it.

Reply:
Er! I'll make a commit to remove these, I forgot.